Automattic/jetpack
# Status: Add WordPress.com Simple helper function

Makes it easier to check if a site is on the WordPress.com platform at all by adding checks for Simple or for Simple/WoA.

#### Changes proposed in this Pull Request:

* Adds is_wpcom_simple
* Adds is_wpcom_platform

#### Other information:

- [x ] Have you written new tests for your changes, if applicable?

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Using WP CLI in Jetpack, run `(new \Automattic\Jetpack\Status\Host)->is_wpcom_simple` and confirm false.
* Same on Atomic and run `(new \Automattic\Jetpack\Status\Host)->is_wpcom_platform` and confirm true.
* Same on WP.com Simple and confirm the above and is_wpcom_simple returns true.
